plugins/default/about.js

```diff
@@ -6,7 +6,7 @@ var about = {
     permission: ['isListener']
  },
  onCall: function(request) {
-    request.sendChat('This broadcast is currently running "Grooveshark Broadcast Bot" ' + GLOBAL.GSBOTVERSION + ', created by grooveshark.com/uman42 ~ github.com/UlysseM/GSBOT/');
+    request.sendChat('This broadcast is currently running "Grooveshark Broadcast Bot" ' + GLOBAL.GSBOTVERSION + ', created by grooveshark.com/uman42 and grooveshark.com/pironic, which is open source! Got a feature or code fix? Submit it to our open repository via pull request at github.com/UlysseM/GSBOT/');
```

Comment:
Mmmmm, technically untrue.
I could so something that parses the plugins, grab all the 'author' names, and display the name automatically, so that contributors that have their code running will appear in the broadcast.